src/components/inputs/select/dropdown.component.tsx

```diff
@@ -46,16 +47,20 @@ const Dropdown: React.FC<FormFieldInputProps> = ({ field, value, errors, warning
   ) : (
     !field.isHidden && (
       <div className={styles.boldedLabel}>
+  
         <Layer>
           <DropdownInput
             id={field.id}
             titleText={<FieldLabel field={field} />}
             label={t('chooseAnOption', 'Choose an option')}
-            items={field.questionOptions.answers
-              .filter((answer) => !answer.isHidden)
-              .map((item) => item.value || item.concept)}
-            itemToString={itemToString}
-            selectedItem={value}
+            items={[
+              { value: '', label: t('selectAnOption', 'Select an option') },
```

Comment:
A better way of doing this is augmenting the question answers from [the schema transformer](https://github.com/openmrs/openmrs-esm-form-engine-lib/blob/main/src/transformers/default-schema-transformer.ts#L137-L166).